ci-operator/step-registry/openshift/e2e/test/hypershift-qe/mgmt/openshift-e2e-test-hypershift-qe-mgmt-chain.yaml

```diff
@@ -8,8 +8,6 @@ chain:
   env:
     - name: TEST_FILTERS
       default: "~ChkUpgrade&;~NonPreRelease&;~DisconnectedOnly&;~MicroShiftOnly&;HyperShiftMGMT&"
-    - name: TEST_TIMEOUT
-      default: "35"
```

Comment:
so the default is not time limit in the ginkgo ?

Reply:
this is a remedy for https://github.com/openshift/release/pull/60323#discussion_r1914294530